maven/lib/dependabot/maven/update_checker/version_finder.rb

```diff
@@ -203,8 +203,18 @@ def fetch_dependency_metadata(repository_details)
         rescue URI::InvalidURIError
           nil
         rescue Excon::Error::Socket, Excon::Error::Timeout,
-               Excon::Error::TooManyRedirects
-          raise if central_repo_urls.include?(repository_details["url"])
+               Excon::Error::TooManyRedirects => e
+
+          if central_repo_urls.include?(repository_details["url"])
+            response_status = response&.status || 0
+            response_body = if response
+                              "RegistryError: #{response.status} response status with body #{response.body}"
+                            else
+                              "Registry error: #{e.message}"
```

Comment:
Should this also be `RegistryError`?
```suggestion
                              "RegistryError: #{e.message}"
```

Reply:
good catch, fixed.